source/assets/stylesheets/_landing.scss

```diff
@@ -9,16 +13,41 @@
 
   background: image-url("hero-header.png") left 0 bottom 1.5em no-repeat;
   height: 460px;
+
+  .show-on-small {
+    display: none;
+  }
+
+  @include media($small-screen) {
+    background: none;
+    .show-on-large {
+      display: none;
+    }
+    .show-on-small {
+      display: block;
+    }
+  }
 }
 
 .event-name {
   font-size: modular-scale(6);
   font-weight: 700;
   margin-bottom: 0;
+
+  @include media($small-screen) {
+    font-size: modular-scale(5);
+    text-align: center;
+    padding: 40px 0 20px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `40px 0 20px` instead of `40px 0 20px 0`